Core/Src/main.c

```diff
@@ -95,6 +104,34 @@ int main(void)
   /* USER CODE BEGIN WHILE */
   while (1)
   {
+	  uint8_t transmit_data[] = {1, 128, 0};	//MCU 1st transmission data: 00000001; MCU 2nd transmission data for single ended CH0: 10000000; 3rd transmission data "don't care" bits
```

Comment:
correct transmit data. I would suggest changing your `128` to hexadecimal form `0x80` which is more commonly used to represent data in firmware development. Also, a `const` can be added to this variable since it is not meant to be changed